sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -85,6 +85,12 @@
       <artifactId>reactor-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.0.1</version>
```

Comment:
@mssfang you missed the version tag here. I can update this with my current PR which is adding additional changes but this should have had a tag on the version line as follows:
<!-- {x-version-update;com.azure:azure-identity;dependency} -->

Reply:
Sounds good to me. Please update it in your PR. Thank you Jim.